### PR TITLE
Implemented Damping Sphere

### PR DESCRIPTION
**Pull request by Zzooouhh**
Not implemented into the setlist yet, because of having an unknown collector number.

### CODE REVIEW

File: Mage.Sets/src/mage/cards/d/DampingSphere.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        return false;
+    }
+
+    @Override
+    public boolean checksEventType(GameEvent event, Game game) {
+        if (event.getType() == EventType.TAPPED_FOR_MANA) {
+            ManaEvent manaEvent = (ManaEvent) event;
+            Mana mana = manaEvent.getMana();
+            return mana.count() > 1;
+        }
```
Comment (JRHerlehy) [inline Mage.Sets/src/mage/cards/d/DampingSphere.java]: Is there are reason you're doing the applies logic in the check event type method?

Comment (Zzooouhh) [reply Mage.Sets/src/mage/cards/d/DampingSphere.java]: Not really, it should fit in either. Does it make a difference in this case?

Comment (JRHerlehy) [reply Mage.Sets/src/mage/cards/d/DampingSphere.java]: Not that I am aware of at this time. Just seems like something SonnarQube would call a code smell is all.

Comment (LevelX2) [reply Mage.Sets/src/mage/cards/d/DampingSphere.java]: I added the checksEventType method some time ago to get a fast precheck to exclude continous effects for further checking during the ApplyEffects method. 

So it makes sense at least for some uniformity to only check the event type there, although the functional outcome is the same.

Comment (Zzooouhh) [reply Mage.Sets/src/mage/cards/d/DampingSphere.java]: Switched mana.count() checking into applies method.

**Pull request merged**